python/pythonmonkey/builtin_modules/timers.js

```diff
@@ -10,14 +10,39 @@ const {
   cancelByTimeoutId
 } = internalBinding('timers');
 
+/**
```

Comment:
What do you think about having this inherit from Number instead? Something like 

```javascript
class TimerId extends Number
{ 
  constructor()
  {
    super();
  }
  unref()
  {
    console.log('hello');
  }
}
```

Reply:
We probably don't need the methods inherited from `Number.prototype` such as [`.toExponential()` ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toExponential)or [`.toFixed()` ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed)

Reply:
The `[Symbol.toPrimitive]()` method will automatically coerce it into a `number` everywhere a `number` is expected.

See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol/toPrimitive